packages/core/src/components/IconLink/IconLink.tsx

```diff
@@ -30,15 +31,15 @@ const useStyles = makeStyles({
   },
 });
 
-export const IconLink = ({
-  href,
-  text,
-  Icon,
-}: {
-  href: string;
-  text?: string;
-  Icon?: IconComponent;
-}) => {
+export const IconLink = (
```

Comment:
Unsure if we want to move this to the core components tbh 😅. Might make more sense to just add an icon prop to the existing link component for example.

Could avoid the debate by duplicating things instead

Reply:
At a glance, it looks like augmenting the existing link component in core would mean shifting from it being a pretty plain wrapper around material-ui's link to needing possibly a grid structure (which IconLink has). So may be mission creep. I think I will duplicate it for now.